### ISSUE TITLE
.d.ts fix

### ISSUE THREAD

**Issue by tripodsgames** [lukeed/klona#21]
I use strict typescript.
When I use klona/full I get the following error:
``
Could not find a declaration file for module 'klona/full'. 'node_modules/klona/full/index.js' implicitly has an 'any' type.
Try `npm install @types/klona` if it exists or add a new declaration (.d.ts) file containing `declare module 'klona/full';`
``

I fixed it by creating a custom definition file, but it would be nice if it was already defined.
```
declare module 'klona/full' {
    export function klona<T>(input: T): T;
}
```

**Comment by lukeed** [comment]
Ah thanks, I thought it would share the same types file but I guess not. Will fix soon!

**Comment by tripodsgames** [comment]
> Ah thanks, I thought it would share the same types file but I guess not. Will fix soon!

if possible, fix this on dequal/lite too, I use your libraries a lot xD

**Comment by lukeed** [comment]
Definitely! Your issue made me think of a few I need to double-check :D 

Appreciate it 🙇

**Comment by lukeed** [comment]
Fixed in `v2.0.2`

**Comment by lukeed** [comment]
And `dequal@2.0.1`
